slither/visitors/expression/constants_folding.py

```diff
@@ -63,25 +64,35 @@ def result(self) -> "Literal":
             # emulate 256-bit wrapping
             if str(self._type).startswith("uint"):
                 value = value & (2**256 - 1)
+        if str(self._type).startswith("byte"):
```

Comment:
mhh would this cause problem for `bytes1`/ `bytes2`/ ...

Reply:
@0xalpharush what do you think? This is part of the constant folding change you made I think.